mod/src/main/java/eu/the5zig/mod/util/IOUtil.java

```diff
@@ -47,7 +47,7 @@ public static String download(String url, boolean post) throws IOException {
 			connection.setConnectTimeout(20000);
 			connection.setReadTimeout(60000);
 			inputStream = connection.getInputStream();
-			return IOUtils.toString(inputStream);
+			return String.valueOf(inputStream);
```

Comment:
Revert, String.valueOf will call InputStream#toString, which is not what we want.